The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-02-14 09:01:04.587 UTC [46] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:01:04.599 UTC [47] LOG:  database system was shut down at 2025-02-14 09:01:04 UTC
2025-02-14 09:01:04.602 UTC [46] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

2025-02-14 09:01:04.957 UTC [46] LOG:  received fast shutdown request
waiting for server to shut down....2025-02-14 09:01:04.957 UTC [46] LOG:  aborting any active transactions
2025-02-14 09:01:04.958 UTC [46] LOG:  background worker "logical replication launcher" (PID 53) exited with exit code 1
2025-02-14 09:01:04.959 UTC [48] LOG:  shutting down
2025-02-14 09:01:04.968 UTC [46] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 09:01:05.066 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 09:01:05.066 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 09:01:05.068 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:01:05.080 UTC [64] LOG:  database system was shut down at 2025-02-14 09:01:04 UTC
2025-02-14 09:01:05.084 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 09:01:11.525 UTC [79] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 09:01:11.525 UTC [79] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 09:01:11.902 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:01:11.902 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_5f9a62d4-83a4-41f8-a796-3f127948706c" WITH (FORCE)
2025-02-14 09:01:12.458 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:01:12.458 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_ef94a6d8-0cca-458e-9651-15b51aae372f" WITH (FORCE)
2025-02-14 09:01:13.020 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:01:13.020 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_f1fffbd6-d771-4657-ba53-229399994c00" WITH (FORCE)
2025-02-14 09:01:13.556 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:01:13.556 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_14731ee2-82d2-4c7a-97f5-e5144303f441" WITH (FORCE)
